README.md

```diff
@@ -75,6 +75,10 @@ In some locales Mercurial uses different encodings for commit messages
 and file names. In that case, you can use `--fe <encoding>` command line
 option which overrides the -e option for file names.
 
+Also Mercurial uses different encodings for branches and tags encoding.
```

Comment:
> Also Mercurial uses ...`
It allows it, but does not do that by default for western locales.

Why not change the previous paragraph say:

In some locales Mercurial uses different encodings for commit messages
and file names. In that case, you can use the `--fe <encoding>` command line
option which overrides the -e option for file names. Likewise the options `--be` and `--te` override the encoding used for branch names and tags respectively.

Reply:
Fixed

Reply:
You forgot to remove the old paragraph starting with "Also ..."m this change also adds trailing whitespace,.